### PR TITLE
Increase External Grader Memory Default

### PR DESCRIPTION
**Pull request by SethPoulsen**

### CODE REVIEW

File: apps/grader-host/src/lib/config.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ const ConfigSchema = z.object({
   jobLogGroup: z.string().default('grading-jobs-debug'),
   reportLoad: z.boolean().default(false),
   reportIntervalSec: z.number().default(10),
-  graderDockerMemory: z.number().default(1 << 30), // 1GiB
+  graderDockerMemory: z.number().default(2 << 30), // 2GiB
   graderDockerMemorySwap: z.number().default(1 << 30), // Same as memory, so no access to swap.
```
Comment (nwalters512) [inline apps/grader-host/src/lib/config.js:30]: You'll need to update this `graderDockerMemorySwap` value too.

File: apps/grader-host/src/lib/config.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ const ConfigSchema = z.object({
   jobLogGroup: z.string().default('grading-jobs-debug'),
   reportLoad: z.boolean().default(false),
   reportIntervalSec: z.number().default(10),
-  graderDockerMemory: z.number().default(1 << 30), // 1GiB
+  graderDockerMemory: z.number().default(2 << 30), // 2GiB
```
Comment (nwalters512) [inline apps/grader-host/src/lib/config.js:29]: This unfortunately does not do what you want it to:

```sh
$ node -e "console.log(2 << 30)"
-2147483648
```

**Review by nwalters512 — CHANGES_REQUESTED** [review]

File: apps/grader-host/src/lib/config.js (hunk 2 above)
Comment (SethPoulsen) [reply apps/grader-host/src/lib/config.js]: ha, that's embarrassing. Should be fixed now.

File: apps/grader-host/src/lib/config.js (hunk 1 above)
Comment (SethPoulsen) [reply apps/grader-host/src/lib/config.js]: done, thanks

**Review by nwalters512 — APPROVED** [review]

**Pull request merged**